benchpress/bqskit_gym/manipulate/test_manipulate.py

```diff
@@ -154,6 +154,31 @@ def result():
 
         assert result
 
+    def test_random_clifford_decompose(self, benchmark):
+        """Decompose a random clifford into
+        basis [rz, sx, x, cz]
+        """
+        qasm_file = Configuration.get_qasm_dir("clifford") + "clifford_20_12345.qasm"
+        circ = Circuit.from_file(qasm_file)
+
+        model = MachineModel(
+            num_qudits=circ.num_qudits,
+            gate_set={SqrtXGate(), XGate(), RZGate(), CZGate()},
```

Comment:
can this be changed to use the gateset in the `default.conf`

Reply:
I see that this line also appears in other tests like `test_QV100_basis_change`:
https://github.com/Qiskit/benchpress/blob/27b978f887d2187827c7792187df7f0d364b7114/benchpress/bqskit_gym/manipulate/test_manipulate.py#L141

Reply:
I'm not sure how to do this. it's probably better to make a separate PR for all the `gate_set` in this file.